src/probability_theory/notation.lean

```diff
@@ -0,0 +1,51 @@
+/-
+Copyright (c) 2021 Rémy Degenne. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Rémy Degenne
+-/
+import measure_theory.function.conditional_expectation
+import measure_theory.decomposition.radon_nikodym
+
+/-! # Notations for probability theory -/
```

Comment:
Since doc-gen doesn't include `notation` commands, can you add a docstring summarizing them?

Reply:
Good point. I will.